bindata/etcd/pod.yaml

```diff
@@ -167,6 +167,10 @@ ${COMPUTED_ENV_VARS}
           --listen-peer-urls=https://${LISTEN_ON_ALL_IPS}:2380 \
           --listen-metrics-urls=https://${LISTEN_ON_ALL_IPS}:9978 ||  mv /etc/kubernetes/etcd-backup-dir/etcd-member.yaml /etc/kubernetes/manifests
     env:
+    - name: GRPC_GO_LOG_VERBOSITY_LEVEL
```

Comment:
etcd wraps gRPC logs so this will not work as expected.

Reply:
Thanks, appreciated the comments. This is very helpful!